content/_data/authors/abayer.adoc

```diff
@@ -3,3 +3,5 @@ name: "Andrew Bayer"
 twitter: abayer
 github: abayer
 ---
+
+Andrew is a core committer to Hudson and the author of numerous plugins. He's the build guy at https://digg.com[Digg], who, by the way, https://jobs.digg.com/[are hiring]!
```

Comment:
hmm there a reason this is going in here? @abayer hasn't been doing the above for a long time

Reply:
His tag line was on a couple articles so I moved it to the authors page for consistency. Most bios are pretty outdated.

We could change it to "Andrew once worked on core"

Reply:
sure